data/hai/hai reveal 1 intro.txt

```diff
@@ -539,8 +541,8 @@ mission "Hai Leaks Response 1"
 			choice
 				`	"That seems practical."`
 				`	"I don't really have an option to refuse, do I?"`
-			`	"Fantastic, you understand the situation," she says, though her tone is much more businesslike than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
-			`	She hands you an officially branded box containing what appears to be a diplomatic ID, which appears to both identify you in human standard and, from context, in Hai.`
+			`	"Fantastic, you understand the situation," she says, though her tone is much more businesslike than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments, then we can begin a dialogue on how best to deal with this current scenario."`
```

Comment:
There's a sentence in the codespell file that is used to make this line pass checks.
That will need to be updated to match this line exactly.

This is because for this particular line "dialogue" is absolutely the correct term, but in general the game needs to check that "dialog" is used correctly in all the usual places it is.